apps/openssl.c

```diff
@@ -392,6 +392,7 @@ int help_main(int argc, char **argv)
 static int do_cmd(LHASH_OF(FUNCTION) *prog, int argc, char *argv[])
 {
     FUNCTION f, *fp;
+    static char alt_buf[256];
```

Comment:
I don't think this needs to be static, I don't see any way this escapes the current scope.